lib/elixir/lib/kernel.ex

```diff
@@ -261,6 +261,8 @@ defmodule Kernel do
   @doc """
   Returns the head of a list. Raises `ArgumentError` if the list is empty.
 
+  It works with improper lists.
```

Comment:
Yeah, that's a good point. It still feels a bit dirty to just say "It works with improper lists" but I can't think of anything better so okay for me :)